daceml/onnx/op_implementations/fpga_implementations.py

```diff
@@ -1006,8 +1006,9 @@ def forward_can_be_applied(node: ONNXOp, state: SDFGState,
         X = in_desc_with_name(node, state, sdfg, "X")
         Y = out_desc_with_name(node, state, sdfg, "Y")
 
-        if Y.veclen != 1:  #NYI
-            return False
+        if Y.veclen != 1:  # if output vectorized must match
+            _, filter_width = node.kernel_shape
+            return X.veclen == Y.veclen or X.veclen == filter_width * Y.veclen  # support reducing vector size proportionally to filter
```

Comment:
Why are you returning immediately? You may skip some of the following checks

Reply:
Should also be adressed now :)